todo_app/app.py

```diff
@@ -1,24 +1,30 @@
 from flask import Flask, render_template, request
-from todo_app.data import session_items
+from todo_app.data import trello_items
 from todo_app.flask_config import Config
+import os
+import requests
 
 app = Flask(__name__)
 app.config.from_object(Config())
 
 @app.route('/')
 def index():
-    items = session_items.get_items()
-    return render_template('index.html', items=items)
+    return render_template('index.html', 
+                            items=trello_items.get_items( app.config['TRELLO_BOARD_KEY'], 
+                                                          app.config['TRELLO_BOARD_TOKEN']) )
 
 @app.route('/addTask', methods=['POST'])
 def add_task():
-    new_task = request.form['TaskName']
-    session_items.add_item(new_task)
+    trello_items.add_item(  app.config['TRELLO_BOARD_KEY'],
+                            app.config['TRELLO_BOARD_TOKEN'],
+                            app.config['TRELLO_TO_DO_LIST_ID'],
+                            request.form['TaskName'] )
```

Comment:
This is starting to get a little unwieldy; we could simplify this a little by for example passing through the entire app config, and letting the function pull out the fields it wants
```python
    trello_items.add_item(  app.config. request.form['TaskName'] )
```

Alternatively, we could even just let the functions that use these fields either read the environment variables directly themselves, or reading the app config using the `current_app` property that Flask provides
```python
# new import
from flask import current_app

def get_items():
    items = []
    app_config = current_app.config
    querystring = {
        "key": app_config['TRELLO_BOARD_KEY'], 
        # or
        "key": os.environ.get('TRELLO_BOARD_KEY')
       ....
```

Reply:
done by breating a Board object and passing that - hope is OK?